components/fieldgroups/profile/profile-consentResultsv2.schema.json

```diff
@@ -0,0 +1,89 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/profile-consentResultsV2",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Consent policies evaluation results map",
+  "type": "object",
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "Consent policies evaluation result for individual",
+  "definitions": {
+    "consent-status": {
+      "title": "string",
+      "description": "current status for a given consent policy",
+      "type": "string",
+      "enum": ["realized", "exited"],
+      "meta:enum": {
+        "realized": "Realized",
+        "exited": "Exited"
+      }
+    },
+    "profile-consentResultsV2": {
+      "type": "object",
```

Comment:
This is a definition. We don't need "type" here. Please remove

Reply:
@mjasrotia Refer to the definition section here: http://json-schema.org/understanding-json-schema/structuring.html
It looks like the entity being referred to is an object (essentially as we are also defining `properties`), so IMO it needs to be present for completeness. LMK.

Reply:
Definitions are only look at as a wrapper by the schema registry. Any object property need to be inside the "properties" section. We can remove the type from the definition here.